### ISSUE TITLE
Solder remaining wires

### ISSUE THREAD

**Issue by theAgingApprentice** [theAgingApprentice/zippy#32]
The motor direction and encoder pins are not soldered but are simply plugged into the header sockets on the Huzzah32 board. Once the issue with the encoders is sorted (see issue #30 and #31 for details) then solder these hires to the dev board.

**Comment by theAgingApprentice** [comment]
Everything except the wires for the reset buttons RGB LED are now soldered in place or screwed down into the Huzzah32 breakout board headers. Waiting on the delivery of a 6 pin header in order to complete this task.

**Comment by theAgingApprentice** [comment]
All wires now soldered into place. One issue with the RGB LED is being dealt with in another issue.
